### DISCUSSION TITLE
kastanjes

### DISCUSSION THREAD

**Discussion by giscus** [Roald87/roaldin.ch#49] [Announcements]
# kastanjes

Kastanjes zijn hier een populaire lekkernij. Zodra de herfst begint, zijn er op straat kraampjes waar je heisse Maroni (warme kastanjes) kunt kopen. Daarnaast worden kastanjes ook vaak geserveerd bij wildmaaltijden of er wordt Vermicelles van gemaakt. Vermicelles is een spaghettivormige puree gemaakt van kastanjes, met suiker en room. Christine Zenino from Chicago, US, CC BY 2.0, via Wikimedia Commons Kastanjebomen vind je veel in het Italiaans-sprekende gedeelte van Zwitserland: Ticino. Wanneer de kastanjes in de herfst rijp zijn en uit de bomen vallen, liggen velden en paden er vol mee. Veel mensen trekken er dan op uit om de kastanjes te verzamelen en deze ‘s avonds te roosteren. Een leuke bezigheid in de herfst, al hebben wij het nooit gedaan. In Nederland is de kastanje een stuk minder populair. Ik kan me wel herinneren dat er in het park bij ons huis veel kastanjebomen stonden, maar deze aten we nooit op. Hoogstens maakten we er op school poppetjes van met tandenstokers.

https://roaldin.ch/kastanjes

**Comment by oom-wim** [comment]
Maar goed dat je er poppetjes van maakte. Dat waren waarschijnlijk de hier veel aangeplante, wilde kastanjebomen of paardenkastanjes: decoratief en giftig!
Inmiddels staan hier (Nl) ook heel wat tamme kastanjebomen in park en natuur, wij rapen ook elk najaar een diepvriesjaarvoorraad bij elkaar. De locaties van de bomen met de grootste kastanjes zijn een goed bewaakt geheim dat we zelfs niet met naaste familie delen.

**Reply by Roald87** [reply]
De tamme kastanjes heb ik wel eens gezien in Nederland. Deze zien er van de buitenkant anders uit en de zaden (?) zijn in mijn herinnering wat kleiner en hoekiger. De kastanjes die ze hier op straat verkopen vind ik meer lijken op de paardenkastanjes. Maar als ik het zo even opzoek, blijken het toch wel de tamme te zijn.
